build-xcframework.sh

```diff
@@ -327,6 +328,15 @@ combine_static_libraries() {
         arch_flags+=" -arch $arch"
     done
 
+    
```

Comment:
Nit: can you remove the extra whitespace on this line (does not show up in github but is visible using a local git diff). You can add [no ci] to the commit to avoid running the full CI again.